### PR TITLE
Chore/occ covid19 manifest update

### PR DESCRIPTION
**Pull request by urvi-occ**
#### Environments

covid19.occ-pla.net - updating manifest.json and adding manifests folder according to the PRC's manifest

#### Description of changes

- Changed versions for micro services
- added the manifest folder with hatchery, fence and modest
- moved hatchery configurations out of the manifest.json

### CODE REVIEW

File: covid19.occ-pla.net/manifests/fence/fence-config-public.yaml [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+LOGIN_OPTIONS:  # !!! remove the empty list to enable login options!
+  - name: 'Login from Google'
+    idp: google
+  - name: 'InCommon Login'
+    idp: fence
+    fence_idp: shibboleth
+    shib_idps: '*'
+  - name: 'ORCID Login'
+    idp: fence
+    fence_idp: orcid
```
Comment (paulineribeyre) [inline covid19.occ-pla.net/manifests/fence/fence-config-public.yaml:109]: these won't work if you don't have the `OPENID_CONNECT` section configured with a fence client

File: covid19.occ-pla.net/manifests/modsec/iis-errors.data [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<h2 style="font:8pt/11pt verdana; color:000000">HTTP 403.6 - Forbidden: IP address rejected<br>
```
Comment (paulineribeyre) [inline covid19.occ-pla.net/manifests/modsec/iis-errors.data:1]: i think WAF is now deployed automatically, so you don't need the `modsec` folder

**Review by paulineribeyre — APPROVED** [review]
a bit outdated (the PRC manifest has been updated since 2021.04)

File: covid19.occ-pla.net/manifest.json [hunk 3]
```diff
@@ -166,7 +115,7 @@
   },
   "ssjdispatcher": {
     "job_images": {
-      "indexing": "quay.io/cdis/indexs3client:2020.03"
+      "indexing": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexs3client:2021.04"
```
Comment (paulineribeyre) [inline covid19.occ-pla.net/manifest.json:118]:
```suggestion
      "indexing": "707767160287.dkr.ecr.us-east-1.amazonaws.com/gen3/indexs3client:2021.07"
```

Comment (urvi-occ) [reply covid19.occ-pla.net/manifest.json]: sorry about that, change is committed now

**Review by paulineribeyre — DISMISSED** [review]

**Review by paulineribeyre — APPROVED** [review]

**Pull request merged**